### PR TITLE
fix: Admin panel fix

### LINKED ISSUE #356: frontend: clean up admin panel
The admin panel is quite sloppy right now. A cleanup is desperately needed.

- [x] Refine layout
- [x] Make layout adaptable to small screens
- [x] Show docker image status

### PR DESCRIPTION
**Pull request by bsilkyn**
There were quite some bugs and fixes requested by my fellow teammates, which I'm trying to correct.

closes #356

### CODE REVIEW

File: frontend/src/assets/lang/app/en.json [hunk 1]
```diff
@@ -285,7 +285,14 @@
         },
         "edit": "Edit",
         "cancel": "Cancel",
-        "save": "Save",
+        "save": {
+            "save": "Save",
+            "error": {
+                "title": "Invalid Safe Operation",
```
Comment (tyboro2002) [inline frontend/src/assets/lang/app/en.json:291]: typo? s/Safe/save/
why does each word start with an uppercase?

File: frontend/src/assets/lang/app/nl.json [hunk 2]
```diff
@@ -281,7 +281,14 @@
         },
         "edit": "Bewerken",
         "cancel": "Annuleer",
-        "save": "Sla op",
+        "save": {
+            "save": "Sla op",
+            "error": {
+                "title": "Ongeldige Opsla Operatie",
+                "detail": "Men kan niet opslaan wanneer geen item is geselecteerd"
```
Comment (tyboro2002) [inline frontend/src/assets/lang/app/nl.json:288]: This does not sound right.

Maybey: "U probeert een item op te slaan zonder dit te selecteeren."

File: frontend/src/assets/lang/app/en.json [hunk 3]
```diff
@@ -285,7 +285,14 @@
         },
         "edit": "Edit",
         "cancel": "Cancel",
-        "save": "Save",
+        "save": {
+            "save": "Save",
+            "error": {
+                "title": "Invalid Safe Operation",
+                "detail": "Trying to save while no item is selected"
```
Comment (tyboro2002) [inline frontend/src/assets/lang/app/en.json:292]: To get the same as the Dutch one

Maybey: "U are trying to save an item without selecting it."

File: frontend/src/assets/lang/app/nl.json [hunk 4]
```diff
@@ -311,7 +318,8 @@
             "private": "Privaat"
         },
         "none_found": "Geen overeenkomende data gevonden.",
-        "loading": "Aan het laden. Wacht een momentje aub."
+        "loading": "Aan het laden. Wacht een momentje aub.",
```
Comment (tyboro2002) [inline frontend/src/assets/lang/app/nl.json:321]: Wacht even aub.

File: frontend/src/assets/lang/app/nl.json [hunk 5]
```diff
@@ -311,7 +318,8 @@
             "private": "Privaat"
         },
         "none_found": "Geen overeenkomende data gevonden.",
-        "loading": "Aan het laden. Wacht een momentje aub."
+        "loading": "Aan het laden. Wacht een momentje aub.",
+        "safeGuard": "Ben je het zeker?"
```
Comment (tyboro2002) [inline frontend/src/assets/lang/app/nl.json:322]: Bent u het zeker?

File: frontend/src/views/admin/UsersView.vue [hunk 6]
```diff
@@ -57,25 +61,36 @@ const roleOptions = computed(() => {
...
     for (let i = 1; i < roles.length; i++) {
         const role: Role = roles[i];
         creators.value[role] = createFunctions.value[i - 1];
     }
 };
-
+/**
+ * FillDestroyers fills a dictionary that links a role to the function that destroys an instance of that role in the backend
+ */
 const fillDestroyers = (): void => {
```
Comment (tyboro2002) [inline frontend/src/views/admin/UsersView.vue:76]: why do this in 2 seperate functions instead of in 1 (fillCreators, fillDestroyers)

**Review by tyboro2002 — CHANGES_REQUESTED** [review]

**Review by EwoutV — APPROVED** [review]
👨‍🚀

**Comment by francisvaut** [comment]
:astonished:

**Comment by Topvennie** [comment]
:1st_place_medal:

**Pull request merged**